apps/build-onchain-apps/contracts/test/SignatureMintERC721.t.sol

```diff
@@ -64,9 +64,6 @@ contract SignatureMintERC721Test is Test {
         assertEq(signatureMintERC721.balanceOf(minter), 0);
         bytes32 messageToSign = signatureMintERC721.getBytesToSign(minter);
 
-        // Expect the FreeMint event to get called
-        vm.expectEmit(true, true, true, true);
-        emit SignatureMintERC721.Mint(minter, 1, 0);
```

Comment:
Question: why is this assertion being removed? Was it causing a problem?

Reply:
This emit event is called  here `signatureMintERC721.freeMint(minter, signature)` , we dont really need to test it seperatelt